### PR TITLE
Add `Corgi Cup VI` tournament article

### PR DESCRIPTION
**Pull request by Bazzadwarf**
The first in a series of PR's that I am currently completing for adding the Corgi Cup tournament series to the wiki.

I have added the info to the main tournament page and created a full entry for Corgi Cup VI. Corgi Cup 1-5 are currently stub articles with a link to the spreadsheet so people can find the info while I create the remaining pieces.

The rules for these tournaments are relatively small by modern tournament standards, but I have left them as is to maintain parity with the original spreadsheets.

I assume that multiple smaller PR's are easier to manage than one colossal PR, but if this isn't the case, please let me know.

### CODE REVIEW

**Review by TPGPL — APPROVED** [review]
cleaned up a bit

should be okay now

File: wiki/Tournaments/CC/6/en.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+  - CC 6
+  - CC VI
+  - CC#6
+---
+
+# Corgi Cup VI
+
+![CCVI logo](img/logo.png)
+
+**Corgi Cup VI** (***CCVI***) was a country-based double-elimination osu! tournament hosted by ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), and ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the sixth installment of the Corgi Cup.
```
Comment (Bazzadwarf) [inline wiki/Tournaments/CC/6/en.md:15]: Shouldn't this be `**Corgi Cup VI** (***CCVI***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), and ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the sixth installment of the Corgi Cup.`

**Review by Bazzadwarf — COMMENTED** [review]
uhhh

File: wiki/Tournaments/CC/6/en.md (hunk 1 above)
Comment (Bazzadwarf) [reply wiki/Tournaments/CC/6/en.md:15]: I guess I'll just change this when pulling in the next PR to save dealing with reopening this.

**Pull request merged**